northstar/src/runtime/console.rs

```diff
@@ -31,12 +31,17 @@ use tokio::{
     pin, select,
     sync::{broadcast, mpsc, oneshot},
     task::{self},
-    time,
+    time::{self, timeout},
 };
 use tokio_util::{either::Either, io::ReaderStream, sync::CancellationToken};
 use url::Url;
 
-const BUFFER_SIZE: usize = 1024 * 1024;
+/// Max length of a single json request line
+const MAX_LINE_LENGTH: usize = 512 * 1024;
+/// Max NPK size
+const MAX_NPK_SIZE: u64 = 512 * 10u64.pow(6);
```

Comment:
nit: how about `1_000_000_u64` instead?

Reply:
wouldn't it be desirable to be able to configure this?

Reply:
If someone needs to configure this, we can add it - I'd wait for a feature request...